docs/en/enterprise-edition/content-collections/dashboards/dashboards-application-security.adoc

```diff
@@ -1,4 +1,4 @@
-== Code Security Dashboard
+== Application Security Dashboard
```

Comment:
@anaghapamidi Match the title with the left navbar heading. WDYT?

Reply:
@divyabhushan - No, I would like to keep as is because the UI is called Code Security Dashboard. And there are future plans for this section.